bin/nib-ls

```diff
@@ -180,6 +184,22 @@ def proc_file(f, opts):
     else:
         row += [ '' ]
 
+    if opts.header_fields:
+        # signals "all fields"
+        if opts.header_fields == '*':
```

Comment:
There is.  Tradeoff between ease of confusion when shell takes over the `*` and likelihood of some future format defining a field called 'all'.